web/discord_relay/queue.go

```diff
@@ -50,6 +51,51 @@ func sendMessage(discord *discordgo.Session, message MessageStruct) {
 	discord.WebhookExecute(WebhookId, WebhookSecret, true, params)
 }
 
+func sendConnect(discord *discordgo.Session, message MessageStruct) {
+	var contentBuilder strings.Builder
+	contentBuilder.WriteString(message.Content)
```

Comment:
instead of using contentBuilder.String() you can just directly pass message.Content